src/ShellClients/PanelWindow.vala

```diff
@@ -119,9 +121,15 @@ public class Gala.PanelWindow : ShellWindow, RootTarget {
         actor.add_action (new DragDropAction (DESTINATION, "multitaskingview-window"));
     }
 
+    public void animate_start () {
+        starting = false;
+        workspace_hide_tracker.recalculate_all_workspaces ();
+    }
+
     protected override double get_hidden_progress () {
         if (starting) {
-            return user_gesture_controller.progress;
+            // Don't show when starting
+            return 1.0;
         }
```

Comment:
```suggestion
```
I would remove this all together then because IMO it's better to allow the user to immediately reveal it instead of trying that and then having it pop in when actually starting since we jump from 1.0 to 0.0

Reply:
Makes sense, thanks for the suggestion!